benchmark_runner/main/environment_variables.py

```diff
@@ -37,6 +37,10 @@ def __init__(self):
         self._environment_variables_dict['elasticsearch_user'] = os.environ.get('ELASTICSEARCH_USER', '')
         self._environment_variables_dict['elasticsearch_password'] = os.environ.get('ELASTICSEARCH_PASSWORD', '')
 
+        # Workaround for Kata CPU offline problem in 4.9/4.10
+        # Set to True to
+        self._environment_variables_dict['kata_cpuoffline_workaround'] = os.environ.get('KATA_CPUOFFLINE_WORKAROUND', '')
```

Comment:
Do you mean we need to enable it for all our workloads run ?

Reply:
We should enable it in the CI runs, and then when we have a proper fix, take it out.

Reply:
For all the workloads, right ?